docker/run.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
Seems redundant, might as well `bundle install` from Dockerfile like other commands.

Reply:
You are right. I let this explicitly on the runtime instead of build as an entrypoint. The idea is mostly that we may need to setup a CICD (Github Actions) -- not sure what is the plan here cc:// @stockholmux but we could make use of the same docker image as a PR validation and then instead of building the image always, we can just use any docker registry e.g. Docker Hub to pull the image and then just mount the site to be served.  That workflow will allow (add checks) if changes will break the site or not.

So to recap, we will have a dedicated docker image that will serve as a base for mounting the site, and then this (`/run.sh`) will be executed on the mounted volume instead of rebuilding the image from the beginning.

Reply:
FWIW re:Github actions the ruby setup is a 2-liner, and Github actions already run in containers. So using Docker for CI with such a simple setup seems like an unnecessary overhead.

Reply:
+1 on enabling some kind of CI to make sure the site builds without markdown errors for example